docs/demos/opc-thermometer-demo.md

```diff
@@ -72,94 +77,87 @@ When mounting certificates is enabled later in the [Running Akri section](opc-th
-5. Now it is time to create our temperature OPC UA Variable. Navigate to the function `CreateAddressSpace` on line 174 of `ReferenceNodeManager.cs` that creates the AddressSpace of the OPC UA Server. To review some terms, [OPC UA specification](https://reference.opcfoundation.org/v104/Core/docs/Part1/3.2/) defines AddressSpace as the "collection of information that a Server makes visible to its Clients", a Node as "a fundamental component of an AddressSpace", and a Variable as a "Node that contains a value". Let create a thermometer Node which has a temperature variable. On line 195, insert the following:
-
-   ```text
-    #region Thermometer
-    FolderState thermometerFolder = CreateFolder(root, "Thermometer", "Thermometer");
-    CreateDynamicVariable(thermometerFolder, "Thermometer_Temperature", "Temperature", DataTypeIds.Int16, ValueRanks.Scalar);
-    #endregion
+3. (Optional) If using security, place the OpcPlc certificate and the CA certificate as below:
+   ```bash
+   plc
+   ├── own
+   │   ├── certs
+   │   │   └── OpcPlc [hash].der
+   │   └── private
+   │       └── OpcPlc [hash].pfx
+   └── trusted
+      ├── certs
+      │   └── SomeCA.der
+      └── crl
+         └── SomeCA.crl
    ```
+4. Open two terminals and run the OPC PLC servers with different ports:
 
-   We selected the `root` folder as the parent of the Thermometer node, which is the `CTT` folder created in line 185. The path to our Thermometer node is Server/CTT/Thermometer, making the NamespaceIndex of the Thermometer Node (and its variables) 2. We care about the `NamespaceIndex` because it along with `Identifier`, are the two fields to a `NodeId`. If you inspect the `CreateDynamicVariable` function, you will see that it creates an OPC UA variable, using the `path` parameter ("Thermometer\_Temperature") as the `Identifier` when creating the NodeID for that variable. It then adds the variable to the `m_dynamicNodes` list. At the bottom of `CreateAddressSpace` the following line initializes a simulation that will periodically change the value of all the variables in `m_dynamicNodes`:
+   If you are **using security**, mount the folder `plc` to `/app/pki/` inside the container:
+   ```bash
+   docker run --rm -it -v path/to/plc:/app/pki -p 50000:50000 --name opcplc1 mcr.microsoft.com/iotedge/opc-plc:latest --pn=50000 --fn=1 --fr=1 --ft=uint --ftl=65 --ftu=85 --ftr=True -aa --ph=your-machine-name
```

Comment:
I wonder if it might be easier to just create a deployment.yaml for these. Then we dont need separate pull and run commands rather just one `kubectl apply`.

Reply:
hi @kate-goldenring, I tried creating a deployment.yaml for these, but ran into tcp connection issues. If we want to simplify the commands, we can remove `docker pull` since it would automatically pull if the image does not exist locally.

Reply:
How about we leave as is and create an issue to create a single deployment in the future? I'd assume the deployment would need a service attached to it.

Reply:
@kate-goldenring @yujinkim-msft using host network instead of cluster network solves the tcp connection issue. This yaml file does the work of deploying 2 opc plc servers:
```apiVersion: apps/v1
kind: Deployment
metadata:
  name: opcplc
  labels:
    app: opcplc
spec:
  selector:
    matchLabels:
      app: opcplc
  template:
    metadata:
      labels: 
        app: opcplc
      name: opc-plc-server
    spec:
      hostNetwork: true
      containers:
      - name: opcplc1
        image: mcr.microsoft.com/iotedge/opc-plc:latest
        ports:
        - containerPort: 50000
        args: ["--pn=50000", "--aa", "--fn=1", "--ft=uint", "--ftl=65", "--ftu=85", "--ftr=True", "--ut", "--sph"]
      - name: opcplc2
        image: mcr.microsoft.com/iotedge/opc-plc:latest
        ports:
        - containerPort: 50001
        args: ["--pn=50001", "--aa", "--fn=1", "--ft=uint", "--ftl=65", "--ftu=85", "--ftr=True", "--ut", "--sph"]
```